docs/tutorials/Tutorial_1_Jason.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# --------------------------------------------------------
+# Distribution statement A. Approved for public release.
+# Distribution is unlimited.
+# This work was supported by the Office of Naval Research.
+# --------------------------------------------------------
+"""Example of PyIRI module validation using Jason TEC.
+
+"""
+
+import datetime
+
+import numpy as np
+import pandas as pd
+
+import PyIRI
+import PyVALION
+
+# In this example, we will create a model output using the PyIRI module
+# You can provide your own model output for the validation
+
```

Comment:
You could make this a scripted example that users could run from the command line, if you like.  To do that, just put all this under `def __main()__:`.  Then, at the bottom of the file, add:

```
if __name__ == '__main__':
    __main__()
```